### DISCUSSION TITLE
some questions about the potential coefficient of gravity field calculated using gravity gradient data

### DISCUSSION THREAD

**Discussion by MaZartmzt** [groops-devs/groops#290] [Q&A]
Hello big guys.
    this is my first time using the Groops ,I need to use the gravity gradient data to calculate the potential coefficient of the gravitational field by least squares configuration.
    So how can I use the functionality in Groops to extend the SGG data on the GOCE satellite radially down onto the mean orbit sphere, and then grid the observations on the sphere using interpolation.I would appreciate it if you could share the steps

**Comment by tmayerguerr** [comment]
Hi, 

I'm not quite sure what you want to do. A few hints:

To estimate a gravity field from GOCE data you can use [NormalsBuild](https://groops-devs.github.io/groops/html/NormalsBuild.html)->design->[gradiometer](https://groops-devs.github.io/groops/html/observationType.html#gradiometer) and next [NormalsSolverVce](https://groops-devs.github.io/groops/html/NormalsSolverVCE.html). Here no interpolation to a constant radius is needed.

To compute gravity gradients at specific points you can use [SimulateGradiometer](https://groops-devs.github.io/groops/html/SimulateGradiometer.html) or [Gravityfield2Gradients](https://groops-devs.github.io/groops/html/Gravityfield2Gradients.html).

Best regards
Torsten
